src/AuthModule/Components/Login/Login.tsx

```diff
@@ -1,77 +1,211 @@
+
   const navigate = useNavigate();
   const {
     register,
     handleSubmit,
     formState: { errors },
-  } = useForm();
+  } = useForm <FormData>();
 
-  const onSubmit = async (data:any) => {
-   
-    await axios.post (`${baseUrl}/v0/admin/users/login`, data)
+  const onSubmit = async (data: FormData) => {
+    // console.log(data);
+    
+    await axios
+      .post(`${baseUrl}/v0/admin/users/login`, data)
       .then((response) => {
         localStorage.setItem("token", response.data.data.token);
-           navigate("/layout");
-           savLoginData();
+        navigate("/layout");
+        savLoginData();
 
         // console.log("token", response.data.data.token);
-        console.log(loginData); 
-        // console.log(baseUrl); 
-
+        console.log(loginData);
+        // console.log(baseUrl);
       })
       .catch((error) => {
-        console.log(error );
-     
+        console.log(error);
       });
   };
- 
+
   return (
     <>
+     
 
-<div className="auth-cotainer vh-100 d-flex justify-content-center align-items-center">
-  <div className="w-50 h-50 bg-info">
-    <form onSubmit={handleSubmit(onSubmit)}>
-      <div className="mb-3 px-5">
-        <label htmlFor="exampleInputEmail1" className="form-label">Email address</label>
-        <input type="email" className="form-control" id="exampleInputEmail1" aria-describedby="emailHelp"
-        {...register("email", {
-          required: "email is required",
-          pattern: {
-            value: /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/,
-            message: "email is not valid ",
-          },
-        })}
-       />
-          <div id="emailHelp" className="form-text">We'll never share your email with anyone else.</div>
-      </div>
-      <div className="mb-3 px-5">
-        <label htmlFor="exampleInputPassword1" className="form-label">Password</label>
-        <input type="password" className="form-control" id="exampleInputPassword1"
-        {...register("password", {
-          required: "password is required ",
-          // pattern: {
-          //   value: /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}$/,
-          //   message:
-          //     "Password must contain at least 8 characters, including upper and lowercase letters, and numbers",
+      <Box
+        sx={{
+          display: "flex",
+          alignItems: "center",
+          height: "100vh",
+          justifyContent: "center",
+          // border: '1px solid',
+          // borderColor: "divider",
+          // borderRadius: 2,
+          // bgcolor: "#cfe8fc",
+          // color: "text.secondary",
+          // "& svg": {
+          //   m: 1,
           // },
-        })}
-        />
-      </div>
-    
-      <button type="submit" className="btn btn-primary px-5">Submit</button>
-    </form>
-  </div>
-</div>
+        }}
+       
+      >
+        <Grid
+          className={Styles.loginContainer}
+          sx={{ bgcolor: "#ffffff" }}
```

Comment:
It is not recommended to have the colors hardcoded, instead we should use colors in the theme , so that if we want to integrate dark/light mode functionality , it will be much easier.